docs/training-debug/index.txt

```diff
@@ -1,343 +1,395 @@
 .. _model-debug:
 
-###########
- Debugging
-###########
+##############
+ Debug Models
```

Comment:
Why not "Debugging Models"?  My thinking is that "Debug Models" does not make it clear that "Debug" is a verb; it could be an adjective the way that it is when you talk about a "debug build" of some C or C++ code.

Reply:
Agree. But headings should not use a gerund (except for Getting Started because it's so ubiquitous). The right title would be 'How to Debug Models' (infinitive form).